src/state/apis/swappers/swappersApi.ts

```diff
@@ -59,31 +48,16 @@ export const swappersApi = createApi({
         // this should never be needed but here for paranoia
         if (!sellAssetUsdRate) throw Error('missing sellAssetUsdRate')
 
-        // We use the sell amount so we don't have to make 2 network requests, as the receive amount requires a quote
-        const isDonationAmountBelowMinimum = getIsDonationAmountBelowMinimum(
-          sellAmountIncludingProtocolFeesCryptoBaseUnit,
-          sellAsset,
-          sellAssetUsdRate,
+        const quoteResults = await getTradeQuotes(
+          {
+            ...getTradeQuoteInput,
+            affiliateBps,
+          },
+          enabledSwappers,
+          selectAssets(state),
         )
 
-        const quoteResults = await Promise.all([
```

Comment:
We don't need to split the quotes into (minimum) "EVM donation swappers" and others now since there's no notion of a minimum per se - simply pass `affiliateBps` as-is, which is itself calculated from the fee curve